include/xtensor/xmanipulation.hpp

```diff
@@ -262,6 +280,101 @@ namespace xt
 
     /// @endcond
 
+    /*****************************
+     *  swapaxes implementation  *
+     *****************************/
+
+    namespace detail
+    {
+        template <class S>
+        inline S swapaxes_perm(std::size_t dim, std::ptrdiff_t axis1, std::ptrdiff_t axis2)
+        {
+            std::size_t const ax1 = normalize_axis(dim, axis1);
+            std::size_t const ax2 = normalize_axis(dim, axis2);
```

Comment:
formatting `const` should be before the type (to be consistent with the rest of the library).